Altis_Life.Altis/core/functions/fn_loadGear.sqf

```diff
@@ -1,87 +1,44 @@
 #include "..\..\script_macros.hpp"
 /*
     File: fn_loadGear.sqf
-    Author: Bryan "Tonic" Boardwine
+    Author: Bryan "Tonic" Boardwine
+    Edited: blackfisch
 
     Description:
     Loads saved civilian gear, this is limited for a reason and that's balance.
 */
-private ["_itemArray","_handle"];
-_itemArray = life_gear;
+private _itemArray = life_gear;
 waitUntil {!(isNull (findDisplay 46))};
 
-_handle = [] spawn life_fnc_stripDownPlayer;
+private _handle = [] spawn life_fnc_stripDownPlayer;
```

Comment:
Can just remove this and make it `call` instead

Reply:
when I think about it, that part can be removed completely since setUnitLoadout overrides current loadout and anything not in that loadout will get removed.

Reply:
nevermind, we still need this for side loadouts